server/types.py

```diff
@@ -57,8 +57,8 @@ def of(cls, params: dict, weight: int = 1):
         if map_size_pixels <= 0:
             raise Exception("Map size is zero or negative")
 
-        if map_size_pixels & (map_size_pixels - 1) != 0:
-            raise Exception("Map size is not a power of 2")
+        if map_size_pixels // 64 != 0:
```

Comment:
Don't you mean 
```suggestion
        if map_size_pixels % 64 != 0:
```
Also add a test?